scene/resources/texture.cpp

```diff
@@ -2703,7 +2703,7 @@ void TextureLayered::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_get_data"), &TextureLayered::_get_data);
 
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "flags", PROPERTY_HINT_FLAGS, "Mipmaps,Repeat,Filter,Anisotropic Filter"), "set_flags", "get_flags");
-	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "data", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NOEDITOR), "_set_data", "_get_data");
+	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "data", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NOEDITOR | PROPERTY_USAGE_RESOURCE_NOT_PERSISTENT), "_set_data", "_get_data");
```

Comment:
Is this really the right solution? `RESOURCE_NOT_PERSISTENT` seems like it's intended for resources, and here it's a dictionary. Indeed, the serialization code had to be changed to handle this dictionary.

Wouldn't this work with `PROPERTY_USAGE_INTERNAL`?

Reply:
CC @Ithamar @KoBeWi